pkg/bundle/bundle.go

```diff
@@ -36,7 +36,7 @@ func PackAndPush(ref name.Reference, directory string, options ...remote.Option)
 		return name.Digest{}, err
 	}
 
-	bundleLayer, err := tarball.LayerFromReader(r)
+	bundleLayer, err := tarball.LayerFromReader(r) //nolint
```

Comment:
I think we should aim for explanatory comments whereever we add `nolint` comments.